packages/ffe-buttons-react/src/BaseButton.js

```diff
@@ -87,6 +89,8 @@ BaseButton.propTypes = {
     leftIcon: node,
     /** Icon shown to the right of the label */
     rightIcon: node,
+    /** Dark variant */
+    dark: bool,
```

Comment:
The `dark` props should have a default value of `false`, even though they are falsy by default. Our linting requires strict equals comparisons and checks like `props.dark === false` would then behave unexpectedly.

Reply:
To be honest I started doing that but since most components did not use default variables for bool I removed it againt :-)

Reply:
As agreed on in the designforum meeting I have added default props all places that accept the dark property